web/custom.js

```diff
@@ -0,0 +1,80 @@
+const webcam = document.getElementById('webcam');
+const canvas = document.getElementById('canvas');
+const user = document.getElementById('userForm');
+const loader = document.getElementById('heartLoader');
+const context = canvas.getContext('2d');
+const biggerText = document.getElementById('issaBiggerText')
+const smallerText = document.getElementById('issaSmallerText')
+const constraints = {
+  video: true,
+};
+//Enables webcam
+navigator.mediaDevices.getUserMedia(constraints)
+  .then((stream) => {
+    webcam.srcObject = stream;
+  });
+
+// Checks if "^" Key is pressed
+document.addEventListener("keydown", function (e) {
+  if (e.keyCode == 192) {
+    // Call the toggle function
+    toggleFullScreen();
+  } else if (e.keyCode == 13) {
+    // Draw the video frame to the canvas and make second stage visible.
+    context.drawImage(webcam, 0, 0, canvas.width, canvas.height);
+    user.style.opacity = 0;
+    user.style.transitionDuration = "1s";
+    loader.style.opacity = 1;
+    loader.style.transitionDuration = "1s";
+    smallerText.style.opacity = 0;
+    smallerText.style.transitionDuration = "1s";
+    crypticWow("Ihr Bild wird nun verarbeitet");
+  } else if (e.keyCode == 70) {
```

Comment:
Same about keyCode = 70 ("f" I guess)?

Reply:
This function will be triggered when the REST API sends the data. For now i simulated is when pressing "f"(finished).